src/utils/signer.py

```diff
@@ -0,0 +1,12 @@
+from django.core import signing
+
+
+def encode_signed_value(value):
+    return signing.dumps(value)
+
+
+def decode_signed_value(signed_value, max_age=None):
+    try:
+        return signing.loads(signed_value, max_age=max_age)
+    except signing.BadSignature:
+        return None
```

Comment:
This does not have any further logic than wrapping a call into a `try`/`except`. I suggest moving this into the service implementation, ideally as a private method.